src/CustomDatePickerAndroid/index.js

```diff
@@ -49,9 +49,11 @@ export default class CustomDatePickerAndroid extends Component {
 
         if (this.props.mode === 'datetime') {
           // Prepopulate and show time picker
-          const timeOptions = this.props.date
```

Comment:
at a first glance the this.props.date check seems redundant
still I left it untouched for consistency